### PR TITLE
Bug/broken monorepo pipelines. support monorepo style with blueprints pipelines

### PR DESCRIPTION
**Pull request by shapirov103**
*Issue #, if available:* #913

*Description of changes:*

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: examples/monorepo/lib/pipeline.ts [hunk 1]
```diff
@@ -0,0 +1,40 @@
+import { App } from 'aws-cdk-lib';
+import * as blueprints from '../../../lib';
+import { logger } from '../../../lib/utils';
+
+export function build(app: App) {
+    const account = process.env.CDK_DEFAULT_ACCOUNT!;
+    const region = 'us-west-2';
```
Comment (elamaran11) [inline examples/monorepo/lib/pipeline.ts:7]: Region is hardcoded.

File: examples/monorepo/lib/pipeline.ts [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+        .codeBuildPolicies(blueprints.DEFAULT_BUILD_POLICIES)
+        .repository({
+            repoUrl: 'cdk-eks-blueprints',
+            credentialsSecretName: 'github-token',
+            targetRevision: 'bug/broken-monorepo-pipelines',
+            path: "examples/monorepo",
+            trigger: blueprints.GitHubTrigger.POLL
+        })
+        .stage({
+            id: 'us-west-2-sandbox',
```
Comment (elamaran11) [inline examples/monorepo/lib/pipeline.ts:36]: region hard coded

File: docs/pipelines.md [hunk 3]
```diff
@@ -209,6 +209,36 @@ const pipeline = blueprints.CodePipelineStack.builder()
             .build(...)
 ```
 
+## Monorepo support
+
+In some cases, customers choose to have their bllueprints defined in subdirectories on a monorepo project. In this scenario, the root of the project is not the root of the CDK pipeline. A single monorepo can also support multiple subprojects each containing its own pipeline.
```
Comment (elamaran11) [inline docs/pipelines.md:214]: typo `bllueprints`

Comment (elamaran11) [reply docs/pipelines.md:214]: `sub directories`

**Review by elamaran11 — COMMENTED** [review]
@shapirov103 Approach looks good, some minor comments.

File: docs/pipelines.md (hunk 3 above)
Comment (shapirov103) [reply docs/pipelines.md]: it is actually subdirectory - one word. Blueprints is fixed.

File: examples/monorepo/lib/pipeline.ts (hunk 1 above)
Comment (shapirov103) [reply examples/monorepo/lib/pipeline.ts]: addressed

File: examples/monorepo/lib/pipeline.ts (hunk 2 above)
Comment (shapirov103) [reply examples/monorepo/lib/pipeline.ts]: addressed

**Review by shapirov103 — COMMENTED** [review]
addressed code review comments

**Review by elamaran11 — COMMENTED** [review]
LGTM. Running the test at my end.

**Comment by elamaran11** [comment]
@shapirov103 The stack ran fine now.

**Review by elamaran11 — APPROVED** [review]

**Pull request merged**